src/EditorFeatures/Test/MetadataAsSource/AbstractMetadataAsSourceTests.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.CodeAnalysis.Editor.UnitTests.MetadataAsSource
     [UseExportProvider]
     public abstract partial class AbstractMetadataAsSourceTests : IAsyncLifetime
     {
-        protected static readonly string ICSharpCodeDecompilerVersion = "7.1.0.6543";
```

Comment:
Not a huge deal, but can't this code just read the version number from the file?